### ISSUE TITLE
Timestamps don't align properly because of the Z character

### ISSUE THREAD

**Issue by paddymul** [buckaroo-data/buckaroo#148]
### Checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://github.com/paddymul/buckaroo) of Buckaroo.

### What type of jupyter notebook were you using (VSCode notebook, google colab, Jupyter Lab, Jupyter notebook).  Select multiple if you can reproduce this in multiple environments.  If other, please add to description.

Jupyter Lab

### Reproducible example

```python
from itables.sample_dfs import get_dict_of_test_dfs
dict_of_test_dfs = get_dict_of_test_dfs()
dict_of_test_dfs["time"]
```

<img width="696" alt="Screen Shot 2023-11-15 at 7 30 32 PM" src="https://github.com/paddymul/buckaroo/assets/40453/5fa4b258-a886-4938-82ab-3774ec38b053">

### Issue description

alignment in the timestamp column is off because the second row doesn't have a `Z` and the third row does.

### Expected behavior

rows without `Z` should be right padded 1 character

### Installed versions

<details>
```
dev version
```
</details>

**Comment by ghost** [comment]
Even third column row one doesn't show proper alignment.rows with 0 as seconds should be left padded 1 character.i.e P2DT0H0M00S

**Comment by paddymul** [comment]
Fixed by #201 for release 0.6.1
